integration/experiment/machine.py

```diff
@@ -36,8 +36,10 @@ def load(self):
 
     def save(self):
         glob_pattern = os.path.join(os.path.dirname(self.path), '*.report')
-        if os.path.exists(self.path) and glob.glob(glob_pattern):
-            raise RuntimeError('<geopm> Machine.save(): function called twice with same path, and reports exist in the same directory.  File exists: {}'.format(self.path))
+
+        # Depending on use case, raising an exception is optional here if machine.py already exists
+        # if os.path.exists(self.path) and glob.glob(glob_pattern):
+        #    raise RuntimeError('<geopm> Machine.save(): function called twice with same path, and reports exist in the same directory.  File exists: {}'.format(self.path))
```

Comment:
Is this change required, or can it be reverted?  If it is required, we should discuss why.

Reply:
TODO: revert changes. 
machine.Machinesave() replaced w/ machine.try_machine(...)